### PR TITLE
Adding tests for graphql

### PR DESCRIPTION
**Pull request by misazr**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 FROM ninech/netbox
 
 RUN ["pip", "install", "graphene-django>=1.0"]
+RUN ["pip", "install", "snapshottest"]
+RUN ["pip", "install", "pytest"]
```
Comment (cimnine) [inline Dockerfile]: Why don't you run `pip install graphene-django>=1.0 snapshottest pytest` ?

File: netbox-graphql/data.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+from circuits.models import CircuitType, Circuit, Provider, CircuitTermination
```
Comment (cimnine) [inline netbox-graphql/data.py]: What purpose does this file have? Is it needed for testing?

Comment (misazr) [reply netbox-graphql/data.py]: Yes, it is needed for making data dump, it for making seeds in tests. https://github.com/ninech/django-netbox-graphql/pull/5/files/d36494041c1bda5f7003becfbece725d6786e65d#diff-9ada9b44ddbe632c13b1563ed6bb691eR6

File: Dockerfile (hunk 1 above)
Comment (misazr) [reply Dockerfile]: ok, I will change it

File: netbox-graphql/data.py (hunk 2 above)
Comment (cimnine) [reply netbox-graphql/data.py]: Shouldn't it be in the `testing` directory then?

Comment (misazr) [reply netbox-graphql/data.py]: it is optional, if you require I can move it there?

Comment (cimnine) [reply netbox-graphql/data.py]: I think it would make more sense.

**Comment by cimnine** [comment]
Have you also seen this comment:

> Wouldn't it make sense to include the snapshots of 'snapshottest'?

**Comment by cimnine** [comment]
It looks like https://github.com/ninech/django-netbox-graphql/pull/5/commits/96756ff2d013a325ac3399abf314c4899552b183 only removed the `data.py`, instead of moving it. Could you check that?

**Comment by cimnine** [comment]
Otherwise it looks good now.

**Comment by misazr** [comment]
data.py is moved from root folder to testing folder...

**Comment by cimnine** [comment]
If I checkout the branch on my machine, the file is deleted. This is also what the commit says.
Can you check, if `git status` shows anything?

**Comment by misazr** [comment]
Some how git didn't commit that file

**Comment by cimnine** [comment]
How do I run the tests? Could you add that to the README?

**Comment by misazr** [comment]
updated readme

**Pull request merged**